### ISSUE TITLE
Uncaught Expection: Object - using msgraph-sdk-javascript (React)

### ISSUE THREAD

**Issue by AriPal** [microsoftgraph/msgraph-sdk-javascript#245]
### Goal:
Get user details using Microsoft Graph API without user authentication, is this the right approach? 

### Research
So I've followed the steps shown on [msgraph-sdk-javascript](https://github.com/microsoftgraph/msgraph-sdk-javascript) for creating an instance of ImplicitMSALAuthenticationProvider in node environment. 

To be honest, I'm not 100 % sure what it means with "in node environment",  the server-side? I'm using React so it may be the wrong steps, maybe "browser environment" is the correct approach?

Anyway, so I'm trying to get user details but it returns `Uncaught Exception: Object`. I'm guessing there are more steps to follow which is not shown in this approach?

Here's the whole code: 
```js
import { UserAgentApplication } from "msal";
import { ImplicitMSALAuthenticationProvider } from "../../../node_modules/@microsoft/microsoft-graph-client/lib/src/ImplicitMSALAuthenticationProvider";
import "isomorphic-fetch";
import { Client } from "@microsoft/microsoft-graph-client";
const msalConfig = {
	auth: {
		clientId: "XXXX",
		// redirectUri: "http://localhost:3000/",
	},
};
const graphScopes = ["user.read"]; // An array of graph scopes
const msalApplication = new UserAgentApplication(msalConfig);
const authProvider = new ImplicitMSALAuthenticationProvider(msalApplication, graphScopes);

// Initialize authentication provider
const opts = {
	authProvider, // An instance created from previous step
};
const client = Client.initWithMiddleware(opts);

// Request users 
const getUsers = async () => {
    try {
        let userDetails = await client.api("/me").get();
        console.log(userDetails); // returns Uncaught Exception: Object
    } catch (error) {
        throw error;
    }
}
getUsers();
```

To summarize, what I really need is to test some data from Microsoft Graph API without having to use the authentication process through login popup etc. Here's a [guide ](https://medium.com/@fiqriismail/how-to-get-an-access-token-for-microsoft-graph-api-using-node-js-258723f29cc6) similar to what I want to achieve.
